contracts/StrategyVersionRegistry.vy

```diff
@@ -0,0 +1,85 @@
+# @version 0.2.12
+governance: public(address)
+pendingGovernance: public(address)
+
+event NewGovernance:
+    governance: address
+
+event NewPendingGovernance:
+    governance: address
+
+event StrategyRegistered:
+    strategy: address
+    name: String[64]
+    apiVersion: String[8]
+
+event Cloned:
+    clone: address
+
+interface Strategy:
+    def name() -> String[64]: view
+    def apiVersion() -> String[8]: view
+    def initialize(params: Bytes[128]): nonpayable
```

Comment:
You can make this larger for future use (this is a dynamic variable, so this means "up to 256 bytes")
```suggestion
    def initialize(params: Bytes[256]): nonpayable
```